### DISCUSSION TITLE
Reliability/consistency of "chia plots check"

### DISCUSSION THREAD

**Discussion by gromran** [Chia-Network/chia-blockchain#6503] [General]
First of all: I have not yet understood exactly what the check does. How can more than 30 proofs be found with an n-value of, for example, 30?

And I once took the trouble to check a few arbitrarily chosen plots with different n-values.
As long as n is the same, the same proof number always comes out for a plot, so that seems to be stable. But different n-values sometimes give very different proofs found:
![plotcheck](https://user-images.githubusercontent.com/2971257/120917320-ed227680-c6ae-11eb-8a65-cb9633199da8.JPG)
This is still quite consistent with individual plots. For others, the ratio across the various n-values is very different. How does that come about?

**Comment by loppefaaret** [comment]
First off, the challenge from `chia plots check` is static. Meaning if you run -n 30 10 times against the same file, it should give the same result everytime.
Imagine that the check issues a challenge with a diff matching the k size of the plot, so it on average should return -n amount of proofs.
Adding in the cryptographic variances, this makes the values swing as you can see - a plot might be lucky against the first 10 or even 50 challenges, but once you get up to 1000, the variance starts zeroing out.

`chia plots check` is meant as a tool for verifying a plot. Not for measuring how "lucky" it is - as you are only running it against one single challence - it defaults to -n 30 - and if you see a plot run less than 0.7, you target that plot with `-g` and run a more thorough check.

**Reply by qpirsel** [reply]
> `chia plots check` is meant as a tool for verifying a plot. Not for measuring how "lucky" it is - as you are only running it against one single challence - it defaults to -n 30 - and if you see a plot run less than 0.7, you target that plot with `-g` and run a more thorough check.

Then the given result as a quality number is really missleading. And even more because one would except that the more n-values are used the results should converge without those "huge" ups and downs seen in many of his results for a given file.
